### PR TITLE
Add serializer options to fix Goldilocks benchmark.

### PR DESCRIPTION
**Pull request by mitchdenny**
Addresses #1776

Added a serializer context. This should enable the app to respond with a valid JSON payload.

### CODE REVIEW

File: src/Goldilocks/Goldilocks.csproj [hunk 1]
```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+    <PackageReference Include="System.Text.Json" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
```
Comment (sebastienros) [inline src/Goldilocks/Goldilocks.csproj:10]: It's part of net8.0, shouldn't be necessary

**Comment by sebastienros** [comment]
Can you confirm there are no more bad requests with these changes?

File: src/Goldilocks/Goldilocks.csproj (hunk 1 above)
Comment (mitchdenny) [reply src/Goldilocks/Goldilocks.csproj]: Fixed the unnecessary dependency.

**Comment by mitchdenny** [comment]
> Can you confirm there are no more bad requests with these changes?

Yes, I just verified again on `8.0.100-alpha.1.22608.10`.

**Pull request merged**